src/include/utime.h

```diff
@@ -220,7 +220,7 @@ class utime_t {
       // raw seconds.  this looks like a relative time.
       out << (long)sec() << "." << std::setw(6) << usec();
     } else {
-      // localtime.  this looks like an absolute time.
+      // UTC time.  this looks like an absolute time.
```

Comment:
why do you think it's UTC time? `utime_t` does not assume or enforce its callers to use UTC or localtime, IIUC. so probably, it would be better to just remove "localtime." from this comment.

Reply:
i agree with you, it's better to remove "localtime. "